pyproject.toml

```
@@ -0,0 +1,75 @@
+[project]
+name = "dmod"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.12"
```

So this is the only thing that give me pause.  The previous project standard was to maintain compatibility with currently supported Python versions.  I think we could bend that rule and remove 3.9, given how close it is to EOL, but 3.10 and 3.11 still each have a decent amount of time left.   

We could discuss changing the project standard too, but I suspect unless something in this compels us to move to >= 3.12, it's simpler just to make a minor adjustment in this PR.

```suggestion
requires-python = ">=3.10"
```